Sources/ClientRuntime/Networking/Http/CRT/CRTClientEngine.swift

```diff
@@ -101,37 +101,49 @@ public class CRTClientEngine: HttpClientEngine {
     ) throws -> HTTPRequestOptions {
         let response = HttpResponse()
         let crtRequest = try request.toHttpRequest()
-        let streamReader: StreamReader = DataStreamReader()
+        let stream = BufferedStream()
 
         let makeStatusCode: (UInt32) -> HttpStatusCode = { statusCode in
             HttpStatusCode(rawValue: Int(statusCode)) ?? .notFound
          }
 
         let requestOptions = HTTPRequestOptions(request: crtRequest) { statusCode, headers in
+            self.logger.debug("headers were received")
             response.statusCode = makeStatusCode(statusCode)
             response.headers.addAll(headers: Headers(httpHeaders: headers))
         } onResponse: { statusCode, headers in
+            self.logger.debug("header block is done")
```

Comment:
What is the difference between this callback and the one above? 
One is "headers were received" and the other is "header block is done". Can we provide a better log string for the latter? I'm unsure what a "header block" is.

Reply:
https://github.com/awslabs/aws-crt-swift/blob/1279ecd11c952f75006c44144986a22fa0fb630e/Source/AwsCommonRuntimeKit/http/HTTPRequestOptions.swift#L8

updated some logs to make it clearer.